proto/content_analysis/sdk/analysis.proto

```diff
@@ -37,6 +37,7 @@ enum AnalysisConnector {
   FILE_ATTACHED = 2;
   BULK_DATA_ENTRY = 3;
   PRINT = 4;
+  FILE_TRANSFER = 5;
```

Comment:
Please leave a comment above this value explaining that this connector is not yet implemented in the SDK but the value is present here to remain consistent with the equivalent chromium code.